backend/clickhouse/sessions.go

```diff
@@ -53,7 +53,7 @@ var fieldMap map[string]string = map[string]string{
 	"browser":           "Browser",
 	"visited_url":       "VisitedURL",
 	"timestamp":         "Timestamp",
-	"secure_id":         "ErrorGroupSecureID",
+	"secure_id":         "SecureID",
```

Comment:
@mayberryzane do you see any issues with this change? Thinking this key is only being used for errors, not sessions, but wanted to double-check with you before changing. Also wondering if we should consider breaking up `fieldMap` to a `sessionFieldMap` and `errorFieldMap`.

Reply:
I dug around and saw the `ErrorGroupSecureID` was introduced in https://github.com/highlight/highlight/pull/6492/files#diff-3ed68c77331af5393b04d063f3a44152d8c2928410c0fb9627dd256adc4ae878R51. I don't believe this was ever working, so I don't see any harm in migrating this key to `SecureID`. I think I'll hold off on refactoring the field maps for now, but definitely something we could do in the future to eliminate some confusion here.